app/views/deployments/recent.html.erb

```diff
@@ -13,5 +13,29 @@
   title: "Recent deployments"
 } %>
 
+<%= form_tag(activity_path, method: :get) do %>
+  <div class="govuk-form-group">
+    <% environment_select = Application::ENVIRONMENTS_ORDER.map { |e| e.capitalize } %>
```

Comment:
I think you could remove this local variable, since it's not used anywhere else(?)

```suggestion

```